chapters/annotations.tex

```diff
@@ -1272,6 +1335,8 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 \end{lstlisting}
 \end{example}
 
+
+
```

Comment:
I guess one empty line should be sufficient here?
```suggestion

```

Reply:
Ok.